xml/System.Collections.Immutable/ImmutableDictionary`2+Builder.xml

```diff
@@ -730,8 +731,8 @@
         <ReturnType>System.Collections.Generic.ICollection&lt;TKey&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets an <see cref="T:System.Collections.Generic.ICollection`1" /> containing the keys of the <see cref="T:System.Collections.Generic.IDictionary`2" />.</summary>
```

Comment:
I presume this is a code comment convention, but using "an" instead of "a" here is because we refer to the type name as opposed to the fully qualified name?

Reply:
In MS Docs, you should see "Gets an ICollection containing..." with the word ICollection enclosed in a hyperlink to the type's page.  So "an" is correct.

Reply:
You know? Ideally, we should be using natural language, since the type will already be shown next to the return value. I'll submit the suggestions so you see what I mean.

Reply:
```suggestion
        <summary>Gets a collection containing the keys of the generic dictionary.</summary>
```